app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt

```diff
@@ -45,6 +45,9 @@ object SaveClicked: EditPatientEvent() {
   override val analyticsName = "Edit Patient Entry:Save Clicked"
 }
 
+// This is NOT an UiEvent, hence doesn't require analytics
```

Comment:
I'm wondering if we should still push this for better observability of what is happening though. All events need not necessarily be UI events, right? It could just be a signal that something of significance happened.